.pre-commit-config.yaml

```diff
@@ -17,10 +18,10 @@ repos:
     - id: end-of-file-fixer
     - id: trailing-whitespace
 
-  - repo: https://github.com/asottile/setup-cfg-fmt
-    rev: v1.17.0
-    hooks:
-    - id: setup-cfg-fmt
+#  - repo: https://github.com/asottile/setup-cfg-fmt
```

Comment:
why comment out?

Reply:
Ah right - I commented because locally it does not work, and I forgot that I had disabled it. Do you know how to fix this problem?

```
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/Users/pizzi/.cache/pre-commit/repozh06y21w/py_env-python3/bin/python', '-mpip', 'install', '.')
return code: 1
expected return code: 0
stdout:
    Processing /Users/pizzi/.cache/pre-commit/repozh06y21w
    Collecting identify[license]>=1.3.0
      Using cached identify-2.4.0-py2.py3-none-any.whl (98 kB)
    Collecting ukkonen; extra == "license"
      Using cached ukkonen-1.0.1.tar.gz (4.1 kB)
    
stderr:
        ERROR: Command errored out with exit status 1:
[...]
            File "/Users/pizzi/.cache/pre-commit/repozh06y21w/py_env-python3/lib/python3.8/site-packages/wheel/macosx_libfile.py", line 356, in calculate_macosx_platform_tag
              assert len(base_version) == 2
          AssertionError
          ----------------------------------------
          ERROR: Failed building wheel for cffi
        ERROR: Failed to build one or more wheels
[...]
 subprocess.CalledProcessError: Command '['/Users/pizzi/.cache/pre-commit/repozh06y21w/py_env-python3/bin/python', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/var/folders/hp/75sjy5v53dngb2t_k9v_bdqh0000gn/T/tmp_57wkqgu', '--quiet', 'cffi>=1']' returned non-zero exit status 1.
[...]
```
Note that I have `cffi==1.14.2` installed in the virtual env without problems (and I'm on macOS)

Reply:
Never mind, I fixed it with `brew install pkg-config libffi`